fs/dafsstream/dafsstream.cpp

```diff
@@ -768,8 +790,8 @@ class CDFUPartReader : public CDaFileSrvClientBase, implements IDFUFilePartReade
             JBASE64_Encode(replyMb.readDirect(cursorLength), cursorLength, cursorInfo, false);
 
             requestTree->setProp("cursorBin", cursorInfo);
-            newReplyMb.rewrite(leadingSpace);
-            handle = sendReadStart(newReplyMb); // new handle
+            newReplyMb.rewrite(leadingSpace); 
```

Comment:
very minor, but there seems to be a whitespace issue here

Reply:
there was a trailing white space, will remove.